mealpy/mealpy.py

```diff
@@ -45,31 +45,34 @@ def login(self, user, password):
 
         return request.status_code
 
-    def get_cities(self):
-        request = self.session.post(CITIES_URL)
-        request.raise_for_status()
-        return request.json()['result']
+    @staticmethod
+    def get_cities():
+        response = requests.post(CITIES_URL)
+        response.raise_for_status()
+
+        result = response.json()['result']
 
-    def get_city(self, city_name):
-        city = next((i for i in self.get_cities() if i['name'] == city_name), None)
```

Comment:
Was this removed deliberately? Seems like still a useful helper as it is used in `get_schedules` and `test_get_cities`.

Reply:
This was deliberate. It's not used anywhere else except `get_schedules`. All it does is list comprehension filtering, which I feel doesn't warrant adding more syntactic sugar around, it's a very common idiom.

I thought hard about use cases and don't forsee any other use besides getting the UUID for schedules. I can see `get_schedule_by_restaurant_name` being reused in the future, so I didn't remove that one. If the need ever arises to de-deduplicate code, it's trivial helper that can be re-added.